src/site/xdoc/getting-started.xml

```diff
@@ -139,9 +139,9 @@ SqlSessionFactory sqlSessionFactory =
         BlogMapper.class). More on this later.
       </p>
     </subsection>
-    <subsection name="Acquiring a SqlSession from SqlSessionFactory">
+    <subsection name="Acquiring an SqlSession from SqlSessionFactory">
```

Comment:
Thanks for your contributing! Your changes is correct!! But I think that this line should be keep because link form existing pages may be broken. 

@harawata WDYT?

Reply:
It depends on the pronunciation [1] and "a SqlSession" should be fine because our documentation is written by the creators of MyBatis. :D

[1] https://www.quora.com/What-is-the-grammatically-correct-way-to-refer-to-the-acronym-SQL-a-SQL-database-or-an-SQL-database

Reply:
> It depends on the pronunciation [1] and "a SqlSession" should be fine because our documentation is written by the creators of MyBatis. :D
> 
> [1] https://www.quora.com/What-is-the-grammatically-correct-way-to-refer-to-the-acronym-SQL-a-SQL-database-or-an-SQL-database

we have many ''an SqlSession" in project, i think we can make the grammar same

Reply:
Well, in that case, let's leave "SqlSession"s as they are.
Both "a" and "an" are not wrong and it does not affect the quality of the documentation (I think).
And, as @kazuki43zoo pointed out, changing 'name' affects external links.